src/components/molecules/PlanRoom/RoomTables/TableComponentsType/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from 'react';
+
+export type TypeItem = {
+  type: string;
+  id: string;
+  top: number;
+  left: number;
+  width: number;
+  height: number;
+  borderRadius: number;
+};
+
+export type TableType = {
+  id: string;
+  left: number;
+  top: number;
+  children: ReactNode;
+  width: number;
+  height: number;
+  borderRadius: number;
+  text: string;
+};
+
+export type TablesType = {
+  [key: string]: { top: number; left: number; text: string; width: number; height: number; borderRadius: number };
```

Comment:
Properties on the right should be extracted to separate type.
I also posted comments about many common properties in those types that you can extract also.

Reply:
It will be separate . Yes, at the moment I have no idea how to do it.